include/openssl/asn1.h

```diff
@@ -141,6 +141,8 @@ DEFINE_STACK_OF(X509_ALGOR)
 # define ASN1_STRING_FLAG_MSTRING 0x040
 /* String is embedded and only content should be freed */
 # define ASN1_STRING_FLAG_EMBED 0x080
+/* String should be parsed in RFC 5280's time format */
+# define ASN1_STRING_FLAG_X509_TIME 0x100
```

Comment:
There is no precedent, all other flags are set internally, so I think we're at liberty to define how to set this particular flag whatever way we wish.  The way it's implemented here seems sane to me.